### ISSUE TITLE
Renaming

### ISSUE THREAD

**Issue by koflerdavid** [rikvdkleij/intellij-haskell#46]
Hello,

I found another problem: In Haskell code when I try to rename a variable, the renaming is not performed within the same `let` statement.

Example:
```haskell
    let (firstState, secondState, thirdState) = (0, 1, 2)
        transitions = transitionTable [((firstState, Just 'a'), [secondState]),
                                       ((firstState, Nothing), [thirdState])]
```

When I rename `secondState` then the occurrence in the second binding is not affected. When I place the cursor in the second occurrence of `secondState` it just renames the first one. Same for `thirdState`. Oddly enough it works for `firstState`...

**Comment by rikvdkleij** [comment]
I tried your case in master branch. It works for all states. 

I hope to release beta soon.

**Comment by rikvdkleij** [comment]
Fixed in latest version, 1.0.0.beta
